rfps/chainlink-and-filecoin.md

```diff
@@ -0,0 +1,65 @@
+
+This joint grant initiative will support up to five teams building superior hybrid smart contract applications powered by tamper-proof file storage and universal connectivity. 
+
+Here are a few examples of possible hybrid smart contract applications, but we encourage all proposals combining Chainlink and Filecoin to apply:
+
+## 1. **Filecoin Miner Insurance**
+Chainlink oracles can inform external smart contracts when a Filecoin miner is unreliable. If the oracles cannot retrieve data from storage or determine the miner is offline, an insurance payout can be automatically issued to policyholders on another blockchain.
+
+## 2. **Data Bounties**
+Smart contracts can pool together funds for the storage of a particular piece of data, as characterized by its unique content identifier (CID). The payment is triggered once the proof that it has been stored on the Filecoin Network is relayed from Chainlink oracles.
+
+## 3. **Perpetual Storage**
+Smart contracts on other blockchains can fund and verify storage on Filecoin thanks to the bidirectional connection of Chainlink. Contracts can be built on networks like Ethereum where anyone can deposit and fund the storage of data at specific intervals on Filecoin.
+
+## 3. **DeFi Data**
+Smart contracts can use Chainlink’s oracles to store extensive amounts of financial data on Filecoin in a tamper-proof manner, which can be delivered on-demand to smart contracts on various blockchains to create more interoperability. 
+
+---
+
+#### Additional Notes
+
+* Proposals may build on Chainlink and Filecoin directly, or use libraries or services that provide abstractions to simplify such as [Estuary](https://estuary.tech/), [Fleek Space Daemon](https://github.com/FleekHQ/space-daemon), [NFT.storage](https://nft.storage/), [Textile Powergate.](https://github.com/textileio/powergate/), et cetera.
+* Projects outside these listed areas are welcome and encouraged to apply for grant funding via the [Filecoin Dev Grants](https://github.com/filecoin-project/devgrants/) program.
+
+#### Proposal Guidelines
+
+We generally fund projects that can be built in a 6-week to 3-month timeframe. Proposals are evaluated based on:
+* The quality of the proposal and near-term value to the Filecoin and Chainlink’s ecosystems
+* Your team’s technical experience, prior open source contributions and interest in diving into how Filecoin works
+* Likelihood that your team will continue to commit to maintaining your project for one year or longer
+
+All work funded via these grants must be open sourced via the MIT license.
+
+#### Proposal Requirements
+
+Proposals should include a value proposition about how it will benefit the Filecoin ecosystem, technical detail about what your team intends to build and evidence your team is capable of creating good, re-usable open source code, compelling product demos and great documentation, and a reasonable technical development plan broken down into milestones. Here's a good sample proposal.
```

Comment:
```suggestion
Proposals should include a value proposition about how it will benefit the Filecoin ecosystem, technical detail about what your team intends to build and evidence your team is capable of creating good, re-usable open source code, compelling product demos and great documentation, and a reasonable technical development plan broken down into milestones. Here's a [good sample proposal](https://github.com/filecoin-project/devgrants/pull/254).
```